specification/cosmos-db/resource-manager/readme.go.md

```diff
@@ -21,6 +22,16 @@ batch:
   - tag: package-2019-08-preview
   - tag: package-2015-04
 ```
+
+### Tag: package-2020-09 and go
+
+These settings apply only when `--tag=package-2020-09 --go` is specified on the command line.
+Please also specify `--go-sdk-folder=<path to the root directory of your azure-sdk-for-go clone>`.
+
+``` yaml $(tag) == 'package-2020-09' && $(go)
+output-folder: $(go-sdk-folder)/services/preview/cosmos-db/mgmt/2020-04-01-preview/$(namespace)
```

Comment:
Since everything in this tag is stable, we cannot put this in the preview subdirectory, and there is a typo
```suggestion
output-folder: $(go-sdk-folder)/services/cosmos-db/mgmt/2020-09-01/$(namespace)
```